composer.json

```diff
@@ -13,45 +13,23 @@
     }
   ],
   "keywords": [
-    "sprint 121",
-    "tao",
+    "TAO Community - Open Source Assessment Software ",
+    "Tao",
     "oat",
+    "open source",
+    "release 3.5",
     "computer-based-assessment"
   ],
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "git@github.com:oat-sa/tao-community.git"
```

Comment:
It seems we're missing to publish it on Packagist so that we can remove this line from here. ;)

Reply:
It's now available on Packagist :) 

https://packagist.org/packages/oat-sa/tao-community